Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN set -x \
 COPY . /app
 
 ARG GIT_HASH
+ENV GIT_HASH=$GIT_HASH
```

Comment:
I doubt this is needed...

Reply:
I copied this from schweizmobil Dockerfiles.